var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -70,7 +70,8 @@ class Podio(CMakePackage):
         deprecated=True,
     )
 
-    depends_on("cxx", type="build")  # generated
+    depends_on("c", type="build")
```

Comment:
Just for clarification: Do we need `depends_on("c", type="build")` if packages are pure c++? Does it make an actual difference in the end?

Reply:
No, you need that only if a C compiler is needed during build. Most of these packages seem to search for a C compiler in CMake. Very likely, they didn't bother changing CMake defaults for projects. In the case of podio (link in the commit):

![Screenshot from 2025-02-10 10-38-01](https://github.com/user-attachments/assets/ae4aa0df-95e6-46da-82d8-b54c61899ae1)

Reply:
:+1: Thanks for the explanation.